app/stores/AuthStore.js

```diff
@@ -138,15 +138,15 @@ export default class AuthStore {
     this.token = null;
 
     // remove authentication token itself
-    Cookie.remove('accessToken', { path: '/' });
+    removeCookie('accessToken', { path: '/' });
 
     // remove session record on apex cookie
     const team = this.team;
     if (team) {
-      const sessions = Cookie.getJSON('sessions') || {};
+      const sessions = JSON.parse(getCookie('sessions') || '{}');
```

Comment:
Both `js-cookie` and `tiny-cookie` automatically run values through `encodeURIComponent` to ensure RFC compliance.

https://www.npmjs.com/package/tiny-cookie#setkey-value-options